genienlp/arguments.py

```diff
@@ -175,6 +175,8 @@ def parse_argv(parser):
                         help='probability of replacing a token with mask for MLM pretraining')
     parser.add_argument('--force_subword_tokenize', action='store_true', default=False,
                         help='force subword tokenization of code tokens too')
+    parser.add_argument('--append_question_to_context_too', action='store_true', default=False,
```

Comment:
The argument name is tad verbose, IMHO

Reply:
This option appends the question to context, but feeds in the original question separately as well, hence "too". What do you suggest?

Reply:
I was going to suggest just "append_question_to_context". Shorter and equally unambiguous.